test/Test/FS.hs

```diff
@@ -128,6 +137,47 @@ prop_numDirEntries (Path dir) isFiles (UniqueList paths) = runSimOrThrow $
     n = length paths
     xs = zip (getInfiniteList isFiles) paths
 
+{-------------------------------------------------------------------------------
+  Corruption
+-------------------------------------------------------------------------------}
+
+data WithBitOffset a = WithBitOffset Int a
```

Comment:
Does this need to be an Int?

Reply:
Doesn't have to be, but if it were a `Word` we would have to convert to an `Int` at some point anyway